### ISSUE TITLE
statvfs is not available on windows

### ISSUE THREAD

**Issue by silentiumsi** [Tinche/aiofiles#171]
statvfs was introduced in the following commit:
https://github.com/Tinche/aiofiles/commit/ec8cc1e27b2ecd6346b24753a1b66fd60f6b7acc

statvfs is not available on Windows as far as I can gather.

```
AttributeError: module 'os' has no attribute 'statvfs'
```

A simple fix may be something like:

```python
if hasattr(os, 'statvfs'):
    statvfs = wrap(os.statvfs)
```

**Comment by Tinche** [comment]
I've just published 23.2.1, let me know!

**Comment by silentiumsi** [comment]
> I've just published 23.2.1, let me know!

This looks to be fixed on Windows 11. At least in the context of https://github.com/AUTOMATIC1111/stable-diffusion-webui

Thanks for handling it so quickly :)
